MasterFastfile/fastlane/.env.default

```diff
@@ -0,0 +1,17 @@
+FL_PROJECT_SIGNING_PROJECT_PATH="./MasterFastfile.xcodeproj"
+GYM_CODE_SIGNING_IDENTITY="iPhone Distribution: The App Business LTD"
+FL_UPDATE_PLIST_APP_IDENTIFIER="com.theappbusiness.MasterFastfile"
+
+FL_HOCKEY_TEAMS=""
+FL_HOCKEY_NOTIFY=0
+FL_HOCKEY_API_TOKEN="011190a883874e9da8d629b3a854df15"
```

Comment:
Currently a dedicated API token for my own personal Hockey account which can be revoked anytime

Reply:
can we make this an env variable in the travis config

Reply:
We can, although that doesn't really help people if they're running locally to test changes. I know you can add env vars to the command when running from Terminal but it's not immediately obvious so we'd need a way to make that clear